README.md

```diff
@@ -66,6 +66,8 @@ SanePatch::Errors::IncompatibleVersion:
   Make sure that the patch at initializers/greeter_patch.rb:8 is still needed and working.
 ```
 
+Setting the `raise_error` keyword argument to `true` will skip the execution of the block but will not raise an error.
```

Comment:
I think you mean setting it to `false` will skip the execution without error?
Also worth mentioning it's `true` by default

Reply:
:facepalm: 

Thank you for catching that, I have pushed a change to the branch correcting the mistake.